### PR TITLE
Get rid of gulp, use lesscpy for LESS stylesheets

### PR DESCRIPTION
**Pull request by njoyard**
The commit in this PR removes the need for gulp and uses [lesscpy](https://github.com/lesscpy/lesscpy) (called from compressor) to compile LESS stylesheets.

It also removes the `libs.less` stylesheet: there is no need to rebuild 3rd party libs LESS stylesheets ourselves when they already ship compiled/minified CSS stylesheets. As a result, the reference to `libs.less` has been replaced by direct individual references to 3rd party CSS files.

**Note :** the gulpfile also served as a livereload trigger. Obviously that's no longer possible after this commit, but it does not seems too big a drawback.

### CODE REVIEW

File: memopol/settings.py [hunk 1]
```diff
@@ -220,9 +220,9 @@
 
 COMPRESS_PRECOMPILERS = (
     # ('text/coffeescript', 'coffee --compile --stdio'),
-    ('text/less', 'lessc {infile} {outfile}'),
-    ('text/x-sass', 'sass {infile} {outfile}'),
-    ('text/x-scss', 'sass --scss {infile} {outfile}'),
+    ('text/less', 'lesscpy {infile}'),
+    # ('text/x-sass', 'sass {infile} {outfile}'),
+    # ('text/x-scss', 'sass --scss {infile} {outfile}'),
```
Comment (Psycojoker) [inline memopol/settings.py]: A number of python portage/binding for sass exist if neede (but I don't know if we use any sass)

https://pypi.python.org/pypi/sass
https://pypi.python.org/pypi/less2sass
https://pypi.python.org/pypi/django-libsass
https://pypi.python.org/pypi/isass

**Comment by Psycojoker** [comment]
Haven't tested but this PR is a great move forward :)

File: memopol/settings.py (hunk 1 above)
Comment (njoyard) [inline memopol/settings.py]: AFAIK we don't use SASS for now, but thanks for the heads up!

**Comment by njoyard** [comment]
I would also like to push this a bit forward:
- remove the need for nodejs altogether by getting rid of bower for 3rd party client libs
- maybe factor installation procedures in a single script (today there are 3 : the instructions in development.rst, the travis scripts and openshift deploy hook).

**Comment by jpic** [comment]
Really awesome, thanks !!!!

And welcome to AUTHORS B)

**Pull request merged**